fixtures/workflow/package.json

```diff
@@ -3,10 +3,12 @@
 	"private": true,
 	"scripts": {
 		"deploy": "wrangler deploy",
-		"start": "wrangler dev --x-dev-env"
+		"start": "wrangler dev --x-dev-env",
```

Comment:
`--x-dev-env` is prolly not necessary anymore since it was turned on by default. But we can def address later